cardea/problem_definition/show_noshow_appointment.py

```diff
@@ -0,0 +1,59 @@
+
+from cardea.problem_definition import ProblemDefinition
+
+
+class PredictingMissedAppointmet (ProblemDefinition):
+    """Defines the problem of missed appointments,
+        whether the patient showed to the appointment or not.
+
+    Attributes:
+        target_label: The target label of the prediction problem.
+        target_entity: The entity name in which it contains the target label.
+    """
+
+    global target_label
+    global target_entity
+
+    target_label = 'status'
+    target_entity = 'Appointment'
+
+    def generate_cutoff_times(self, entity_set):
+        """Generates cutoff times for the predection problem.
+
+        Args:
+            entity_set: FHIR entityset.
+
+        Returns:
+            DataFrame with entity id, cutoff times and target labels.
+
+        Raises:
+            ValueError: An error occurs if the target label contains a missing value.
+            ValueError: An error occurs if the target label does not exist.
+
+        """
+
+        if self.check_target_label(
+                ProblemDefinition,
+                entity_set,
+                target_entity,
+                target_label):
+            if self.check_target_label_values(
+                    ProblemDefinition, entity_set, target_entity, target_label):
+                raise ValueError('Please remove missing values in the target label')
```

Comment:
It would be more instructive if the error you are raising informs the user what `entity` contains missing values and in what column specifically.